app/views/layouts/angular/_generic_form_buttons.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
Aah, yeah, agreed `buttonGroupController` should go..

Ok, if the idea is to replace all of those by all of these, sure, works for now :)